src/client/lib/sentryLogger.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseLogger } from '@/shared/lib/baseLogger';
+import { LogLevel } from '@/shared/model/Logger';
+import * as Sentry from '@sentry/browser';
+
+const getSentryLevel = (level: LogLevel) => {
+  switch (level) {
+    case LogLevel.ERROR:
+      return Sentry.Severity.Error;
+    case LogLevel.INFO:
+      return Sentry.Severity.Info;
+    case LogLevel.WARN:
+      return Sentry.Severity.Warning;
+    default:
+      return Sentry.Severity.Log;
+  }
+};
+
+class SentryLogger extends BaseLogger {
```

Comment:
Would `ClientLogger` make more sense to differentiate between sentry and winston, and where it should be used?

Reply:
have changed both to follow a client/server naming convention